egs/librispeech/ASR/local/display_manifest_statistics.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This file displays duration statistics of utterances in a manifest.
+You can use the displayed value to choose minimum/maximum duration
+to remove short and long utterances during the training.
+
+See the function `remove_short_and_long_utt()` in transducer/train.py
+for usage.
+"""
+
+
+from lhotse import load_manifest
+
+
+def main():
+    path = "./data/fbank/cuts_train-clean-100.json.gz"
+    path = "./data/fbank/cuts_train-clean-360.json.gz"
+    path = "./data/fbank/cuts_train-other-500.json.gz"
+    path = "./data/fbank/cuts_dev-clean.json.gz"
+    path = "./data/fbank/cuts_dev-other.json.gz"
+    path = "./data/fbank/cuts_test-clean.json.gz"
+    path = "./data/fbank/cuts_test-other.json.gz"
+
+    cuts = load_manifest(path)
+    cuts.describe()
+
+
+if __name__ == "__main__":
+    main()
+
+"""
+## train-clean-100
+Cuts count: 85617
+Total duration (hours): 303.8
```

Comment:
Due to data augmentation, i.e., speed perturb with factors 0.9 and 1.1, the total duration is tripled.